Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -24,7 +24,7 @@
 			<key>pfm_default</key>
 			<string>Configures XCreds configuration preferences</string>
 			<key>pfm_description</key>
-			<string>The human-readable description of this payload. This description appears on the Detail screen.</string>
```

Comment:
Let's not make changes to the common payload keys (those stating with `Payload...`) because they are generic and except for their default values should be the same throughout all manifests.